<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Braintux</class>
 <widget class="QMainWindow" name="Braintux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Braintux</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(173, 127, 168)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Braintux</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Braintux is made by modules, so use them.</string>
    </property>
   </widget>
   <widget class="QLabel" name="terminal_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Terminal</string>
    </property>
   </widget>
   <widget class="QLabel" name="whatsapp_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Whatsapp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="terminal_start_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="terminal_stop_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="whatsapp_stop_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="whatsapp_start_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qt_stop_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qt_start_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 53, 102)</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="Line" name="output_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="terminal_status_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="whatsapp_status_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
color: rgb(238, 238, 236)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_status_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 191, 255)</string>
    </property>
    <property name="text">
     <string>Qt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
